packages/service/index.d.ts

```diff
@@ -29,6 +29,42 @@ declare module 'fastify' {
 
 type DefaultGenerator = new () => BaseGenerator.BaseGenerator
 
+export interface StartOptions {
+  listen?: boolean
+}
+
+export interface StackableInfo {
+  type: string
+  version: string
+}
+
+export interface StackableInterface {
+  init: () => Promise<void>
+  start: (options: StartOptions) => Promise<void>
+  stop: () => Promise<void>
+  getUrl: () => string
+  getConfig: () => Promise<object>
+  getInfo: () => Promise<StackableInfo>
+  getDispatchFunc: () => Promise<Function>
+  getOpenapiSchema: () => Promise<object>
+  getGraphqlSchema: () => Promise<string>
+  getMetrics: () => Promise<string>
+  inject: (injectParams: object) => Promise<{
+    statusCode: number
+    statusMessage: string
+    headers: object
+    body: object
+  }>
+}
```

Comment:
I think this should be defined nearby https://github.com/platformatic/platformatic/blob/d1970c49257295cc649a037c128e79103acef6d5/packages/config/index.d.ts#L72.